### PR TITLE
Add Hebrew online books

### PR DESCRIPTION
**Pull request by tomer**
## What does this PR do?
Add Hebrew online books provided by [cyber.org.il](http://cyber.org.il) and [education.gov.il](https://education.gov.il)

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md)
- [X] Search for duplicates.
- [x] Include author(s) and platform where appropriate.
- [X] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction)

## Followup

- Check the output of Travis-CI for linter errors!

### CODE REVIEW

File: books/free-programming-books-he.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+### Index
+
+* [C#](#cs)
+* [Assembly](#assembly)
+* [Python](#python)
+* [ללא תלות בשפה](#ללא-תלות-בשפה)
+  * [מערכות הפעלה](#מערכות-הפעלה)
+  * [רשתות](#רשתות)
```
Comment (davorpa) [inline books/free-programming-books-he.md:8]: Could you adapt this:

```markdown
* [ללא תלות בשפה](#ללא-תלות-בשפה)
  * [מערכות הפעלה](#מערכות-הפעלה)
  * [רשתות](#רשתות)
```

to follow more or less the style of the other listings? I'm not be able to RTL write with numbers.

- 0 - Metalists
  * book
- 1 - Agnostic to language
  * book | section
- Assembly

E.g: https://github.com/EbookFoundation/free-programming-books/blob/master/books/free-programming-books-es.md

I mean, to move it as the first listing section.

If it's not posible, doesn't matter

File: books/free-programming-books-he.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+### Index
+
+* [C#](#cs)
+* [Assembly](#assembly)
```
Comment (davorpa) [inline books/free-programming-books-he.md:4]: - We should preserve order in TOC too :wink:
- Like in other listings use `c-sharp` as anchor alias

```suggestion
* [Assembly](#assembly)
* [C#](#c-sharp)
```

File: books/free-programming-books-he.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+  * [מערכות הפעלה](#מערכות-הפעלה)
+  * [רשתות](#רשתות)
+
+
+### Assembly
+
+* [ארגון המחשב ושפת סף](https://data.cyber.org.il/python/python_book.pdf) – ברק גונן, המרכז לחינוך סייבר (PDF)
+
+
+### <a name="cs">C#</a>
```
Comment (davorpa) [inline books/free-programming-books-he.md:16]: - Usually, an anchor that represent an alias has no content.
 - Also apply rename in TOC: `c-sharp` 

```suggestion
### <a name="c-sharp"></a>C#
```

File: books/free-programming-books-he.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+### ללא תלות בשפה
+
+#### מערכות הפעלה
+
+* [מערכות הפעלה](https://data.cyber.org.il/os/os_book.pdf) – ברק גונן, המרכז לחינוך סייבר (PDF)
+
+
+#### רשתות
+
+* [רשתות מחשבים](https://data.cyber.org.il/networks/networks.pdf) – עומר רוזנבוים, ברק גונן, שלומי הוד, המרכז לחינוך  (PDF)סייבר
```
Comment (davorpa) [inline books/free-programming-books-he.md:35]: Move to top with the purpose of mantaining list alphabetization proposed at TOC code hunk

If it's not possible, ignore it.

**Review by davorpa — CHANGES_REQUESTED** [review]
Take a look at this some minor changes

File: books/free-programming-books-he.md (hunk 1 above)
Comment (tomer) [reply books/free-programming-books-he.md:5]: Done by commit 9e1c741

File: books/free-programming-books-he.md (hunk 4 above)
Comment (tomer) [reply books/free-programming-books-he.md]: Done by commit 9e1c741

File: books/free-programming-books-he.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
```
Comment (davorpa) [inline books/free-programming-books-he.md:20]: @tomer, should this be reword as like in other `ciber` resources? I found a bit strange this glyphs after the (PDF) tag

`עומר רוזנבוים, ברק גונן, שלומי הוד, המרכז לחינוך (PDF)`**`סייבר`**

`המרכז לחינוך` -> center of education
`סייבר` -> ciber

Comment (tomer) [reply books/free-programming-books-he.md]: You right. It caused because the online editor doesn't respect RTL content.

**Review by davorpa — APPROVED** [review]
Ready to merge

**Comment by eshellman** [comment]
This goes in our highlight reel.

**Pull request merged**